src/features/user/Settings/EditProfile/EditProfileForm.tsx

```diff
@@ -296,150 +321,225 @@ export default function EditProfileForm() {
-                  <img
-                    src={getImageUrl('profile', 'thumb', user.image)}
-                    className={styles.profilePicImg}
-                  />
-                  <div className={styles.profilePicOverlay} />
-                  <CameraWhite />
-                </div>
-              ) : (
-                <div className={styles.noProfilePic}>
-                  <Camera />
-                </div>
-              )}
+        <div className={styles.profilePicDiv}>
+          {updatingPic ? (
+            <div className={styles.spinnerContainer}>
+              <div className={styles.spinnerImage}></div>
+            </div>
+          ) : user?.image ? (
+            <div className={styles.profilePic}>
+              <img
+                src={getImageUrl('profile', 'thumb', user.image)}
+                className={styles.profilePicImg}
+              />
+            </div>
+          ) : (
+            <div className={styles.noProfilePic}>
+              <DefaultUserProfileImage />
             </div>
-          </label>
+          )}
+          <div className={styles.profilePicDivButtons}>
+            <div {...getRootProps()} className={styles.uploadProfilePicButton}>
+              <label htmlFor="upload">
+                <input {...getInputProps()} />
+                <Camera />
+                <span>{t('profilePictureButtonLabels.upload')}</span>
+              </label>
+            </div>
+            <button
+              className={styles.deleteProfilePicButton}
+              onClick={(event) => deleteProfilePicture(event)}
+            >
+              <label htmlFor="delete">
+                <Delete color="#828282" />
+                <span>{t('profilePictureButtonLabels.delete')}</span>
+              </label>
+            </button>
+          </div>
         </div>
 
         {type !== 'tpo' ? (
-          <MuiAutoComplete
-            id="profile-type"
-            value={localProfileType}
-            options={profileTypes}
-            getOptionLabel={(option) => (option as ProfileTypeOption).title}
-            isOptionEqualToValue={(option, selectedOption) =>
-              (option as ProfileTypeOption).value ===
-              (selectedOption as ProfileTypeOption).value
-            }
-            renderOption={(props, option) => {
-              const { id, title } = option as ProfileTypeOption;
-              return (
-                <StyledAutoCompleteOption {...props} key={id}>
-                  {title}
-                </StyledAutoCompleteOption>
-              );
-            }}
-            onChange={(event, newType) => {
-              if (newType) {
-                setAccountType((newType as ProfileTypeOption).value);
-                setLocalProfileType(newType as ProfileTypeOption);
+          <EditProfileInputContainer>
+            <StyledInputLabel shrink={false} htmlFor="profile-type">
+              <Typography>{t('fieldLabels.profileType')}</Typography>
```

Comment:
Is there a specific reason to use `Typography`, when just the text would suffice?